cns/client/client.go

```diff
@@ -1023,11 +1023,11 @@ func (c *Client) GetHomeAz(ctx context.Context) (*cns.GetHomeAzResponse, error)
 	return &getHomeAzResponse, nil
 }
 
-// GetEndpoint calls the EndpointHandlerAPI in CNS to retrieve the state of a given EndpointID
-func (c *Client) GetEndpoint(ctx context.Context, endpointID string) (*restserver.GetEndpointResponse, error) {
+// GetEndpoint calls the EndpointHandlerAPI in CNS to retrieve the state of a given containerID
+func (c *Client) GetEndpoint(ctx context.Context, containerID string) (*restserver.GetEndpointResponse, error) {
 	// build the request
 	u := c.routes[cns.EndpointAPI]
-	uString := u.String() + endpointID
+	uString := u.String() + containerID
```

Comment:
Will this affect the existing endpoints ?

Reply:
I don't seem to see this change on my branch anymore-- this is just a variable name change so what actually changes is what we use as the "key"-- as long as it is consistent then what we set is what we get later